### DISCUSSION TITLE
A small question about getting the API Key from local.ai

### DISCUSSION THREAD

**Discussion by AmandaKimo** [louisgv/local.ai#57] [Q&A] [answered]
Hello, I'm new with all this stuff and I was wondering how do I get the API Key from local.ai to window.ai. I already dowloaded a model and runned the server but I still have no idea how to get the key.

**Comment by louisgv** [comment]
There's no key, you just call the local server - see: https://github.com/louisgv/local.ai/issues/30#issuecomment-1597657199

We will have a wiki section soon cc @Regular-Baf

**Reply by louisgv** [reply, marked as answer]
Completed in: https://github.com/louisgv/local.ai/wiki#api-specs
